internal/apigw/apigw_openapi.go

```diff
@@ -175,6 +188,62 @@ func (module *Module) buildOperation(ctx pgsgo.Context, method pgs.Method, mt *m
 	return r, op, components, nil
 }
 
+func routeDotsToCamelCase(s string) (string, error) {
```

Comment:
i think you can use a `pgs` String helper here?

Reply:
https://pkg.go.dev/github.com/lyft/protoc-gen-star#Name.LowerCamelCase

Reply:
also you can use the Route parser here:
https://github.com/ductone/protoc-gen-apigw/blob/main/apigw/v1/http_bridge.go#L58-L92

instead of string splitting?

Reply:
pgs.Name is great, I like that

I don't want to do this in parse route though because it's used in a few other places (including our backend) that relies on the dot syntax to get nested params which is what Anthony was working on for the medium size pr

Reply:
no you don't need to "do this" in parse route, but you can sue parseRoute to... parse it?